### PR TITLE
add asciinema video for BCD getting started tutorial

### PR DESCRIPTION
**Pull request by tnguyen1**

### CODE REVIEW

**Comment by JeremJR** [comment]
retest this please

**Comment by tnguyen1** [comment]
retest this please

File: md/getting_started.md [hunk 1]
```diff
@@ -16,6 +16,19 @@ Then a **BCD Controller container** has to be started on the control host, ie. t
...
+    var file = document.createElement("link");
+    file.setAttribute("rel", "stylesheet");
+    file.setAttribute("type", "text/css");
+    file.setAttribute("href", filename);
+    document.head.appendChild(file);
+}
+loadCSS("./assets/asciinema-player.css");
+</script>
+<asciinema-player src="./bcd/images/${varVersion}/bcd_getting-started.cast" speed="2" poster="data:text/plain,\e[2;3HGetting started with \e[1;34mBonita Continuous Delivery\u001b[0m controller." theme="monokai" title="Getting started with Bonita Continuous Delivery controller" cols="240" rows="32"></asciinema-player>
+<script src="./assets/asciinema-player.js"></script>
```
Comment (educhastenier) [inline md/getting_started.md:29]: where are the .css and .js files? They don't appear in this PR...

Comment (JeremJR) [reply md/getting_started.md:29]: they are already merged into the bonita-doc-site repo https://github.com/bonitasoft/bonita-doc-site/pull/108

**Comment by uguy** [comment]
retest this please

**Comment by uguy** [comment]
This PR should be able to use BCD and Bonita version as vars, to avoid recreate the animation script on every release.
This would require some work in `convertMdToHtml.js` and `taxonomy.js` as well as define vars with full versions in `variables.json`

Need to clarify also if the embedded player (asciinema-player) alone is enough without the script imports (since already in the global site ?)

**Comment by tnguyen1** [comment]
I agree we need to be able to to inject BCD and Bontia versions as variables into the asciinema script. Maybe the new doc-site framework will allow this. I'm suggesting to close this PR and redo it once the migration is done no? cc @uguy

**Comment by uguy** [comment]
I added asciinema support to the new site, so I will try to migrate this PR (and maybe update BCD version)

**Pull request merged**